step.sh

```diff
@@ -1,80 +1,31 @@
 #!/bin/bash
 set -e
 
-# Check that collector_config exists
-if [ ! -f "$collector_config" ]
-then
-  echo "Error: collector_config at $collector_config does not exist"
-  exit 1
-fi
-
-# Check that iglu eixsts
-if [ ! -f "$iglu" ]
-then
-  echo "Error: iglu at $iglu does not exist"
-  exit 1
-fi
-
-# Resolve the latest version if we need to
-echo "Request server version: $micro_version"
-if [[ "$micro_version" == "latest" ]]
-then
-  # See which tag the HTTP request is redirected to (can't use API due to unauthenticated rate limits)
-  micro_version=$(curl -I --silent https://github.com/snowplow-incubator/snowplow-micro/releases/latest | grep -iF location: | sed -E 's/\r$//' | sed -E 's/^.*tag\/(.*)$/\1/')
+httpschema="https"
+interface="snowplow-micro-ios.production.data.global-services.ckpd.co"
```

Comment:
So `interface` and `port` only really made sense when this was dealign with starting the localhost collector, when we move this script into reset-collector, we can get away with just using the input parameter `$collector_url` (which will be the base url), just like check-collector 👍